File: submissions/Utkarsh Saxena/day13_coordinates.c
#include <stdio.h>
int main(){
    int digit,N,EvenDigit=0;
    printf("Enter a number: ");
    scanf("%d", &N);
     printf("Even Digit: ");
     while(N>0){
        digit = N%10;
    if(digit%2 == 0){
        printf("%d ", digit);
        EvenDigit=1;
    }  
    N /= 10;
  }
  if(EvenDigit == 0){
    printf("No Even Digits found");
  }
  return 0;
}

File: submissions/Devang_Shukla/day13_coordinates.py
#Day 13 - Extract Even Digits from a Number

# Take input from the user 
N = input("Enter N:")

#List to store even digit 
even_digit = []

#Iterate through each digit in the number 
for digit in N:
   if digit.isdigit() and int(digit) % 2 == 0:
      even_digit.append(digit)

# Print result based on whether any even digit were found
if even_digit:
   print("Even digit:",' '.join(even_digit))
else:
    print("No even digit found!")     